packages/vaadin-lumo-styles/src/mixins/combo-box-overlay.css

```diff
@@ -0,0 +1,24 @@
+[part='content'] {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 0;
+}
+
+/* When items are empty, the spinner needs some room */
+:host(:not([closing])) [part~='content'] {
```

Comment:
nit: we could probably use `[part='content']` and `[part='overlay']` consistently, not need for `~=` selector.

Reply:
Done.